### ISSUE TITLE
support nodetool verify

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#2207]
Verify (check data checksum for) one or more tables.

example 
```
nodetool verify myks mytable
```

https://issues.apache.org/jira/browse/CASSANDRA-9742

**Comment by tzach** [comment]
@denesb AFAIK this is covered by the exiting sstable tooling.
Is it the case?

**Comment by denesb** [comment]
> @denesb AFAIK this is covered by the exiting sstable tooling. Is it the case?

Yes and no. If you have compressed sstables, `nodetool scrub --mode=VALIDATE` will do the same as verify (but it will still not check the digest, only the per-chunk checksums).
For uncompressed sstables, there is no checksum/digest checked at all.

The closest is `scylla sstable validate-checksums`, which checks both digest and checksums for any sstable.

**Comment by tzach** [comment]
so scrub+validate-checksums >= verify ?

**Comment by denesb** [comment]
> so scrub+validate-checksums >= verify ?

`scylla sstable validate-checksums` >= verify (but its offline)

Once I fix https://github.com/scylladb/scylladb/issues/15693 -- or the part of it which covers scrub --mode=VALIDATE always checking checksums -- `nodetool scrub --mode=VALIDATE` will cover all `nodetool verify` functionality as well. At that point we can introduce it as a simple alias, for compatibility.

**Comment by denesb** [comment]
`nodetool scrub --mode=VALIDATE` now checks both digests and checksums, see https://github.com/scylladb/scylladb/issues/19058.
